docs/source/user-guide/mission-modeler-guide/activity-types/activity-parameters.md

```diff
@@ -0,0 +1,142 @@
+# Activity Parameters
+
+Activity parameters provide the ability to modulate the behavior of an activity instance's effect model.
+Aside from determinining the effects of the activity, these parameters can be used to determine its duration, 
+decomposition into children activities and expansion into commands.
+
+The Merlin annotation processor is used to extract and generate serialization code for parameters of activity types.
+The annotation processor also allows authors of a mission model to create mission-specific parameter types, 
+ensuring that they will be recognized by the Merlin framework.
+See [Advanced Topic: Parameters](../parameters/index) for a thorough explanation 
+of all possible styles of parameter declarations.
+For more information on mission-specific parameter types, see [Value Mappers](activity-mappers.md#value-mappers).
+
+## Parameter Validations
+
+A mission model configuration or activity instance can be validated by providing one or more methods annotated by `@Validation` and `@Validation.Subject`
+The `@Validation` annotation message specifies the message to present to a planner when the validation fails.
+The `@Validation.Subject` annotation specifies which parameter(s) the validation is associated with.
+These associated parameters are reported from Merlin when querying for activity argument validations to couple validations with parameters.
+
+For example:
+
+```java
+@Validation("instrument power must be between 0.0 and 1000.0")
+@Validation.Subject("instrumentPower_W")
+public boolean validateInstrumentPower() {
+  return (instrumentPower_W >= 0.0) && (instrumentPower_W <= 1000.0);
+}
+```
+
+To associate a validation with multiple parameters `{"...", "...", ...}` syntax may be used within the `@Validation.Subject` annotation.
+For example:
+
+```java
+@Validation("instrument powers must be between 0.0 and 1000.0")
+@Validation.Subject({"instrumentPower_W", "instrumentPower_Y"})
+public boolean validateInstrumentPowers() {
+  return (instrumentPower_W >= 0.0) && (instrumentPower_W <= 1000.0) &&
+      (instrumentPower_Y >= 0.0) && (instrumentPower_Y <= 1000.0) &&;
+}
+```
+
+The Merlin annotation processor identifies these methods and arranges for them to be invoked whenever the planner instantiates an instance of the class.
+A message will be provided to the planner for each failing validation, so the order of validation methods does not matter.
+
+## Duration Types
+
+The scheduler (see [Scheduling Guide](../activity-plans/scheduling-guide) places 
```

Comment:
Link needs to be updated: `(see [Scheduling Guide](../activity-plans/scheduling-guide)`. This one probably wants to point to `../../ui-api-guide/scheduling/index.rst`

Reply:
Thanks for finding the correct link! Just pushed this fix.